src/components/Providers/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react'
+import { MuiThemeProvider, Theme as MuiTheme } from '@material-ui/core/styles'
+import { ConnectedRouter } from 'connected-react-router'
+import { Provider } from 'react-redux'
+import { ThemeProvider } from 'styled-components'
+import { Store } from 'redux'
+import { History } from 'history'
+import { Theme } from '@gnosis.pm/safe-react-components/dist/theme'
```

Comment:
I can see some duplication as passed theme is always the same either on tests or main app. Although it makes sense to pass it from the upper level as like then you are passing all providers from the parent component.

Would it make sense to export this type from the safe-react-components library?